doc/guides/dts/index.rst

```diff
@@ -169,8 +172,7 @@ of things will vary based on the board layout and application use.
 Currently supported boards
 **************************
 
-Device tree is currently supported on all ARM targets.  Support for all other
-architectures is to be completed by release 1.11.
+Device tree is currently supported on all targets except posix (boards/posix).
```

Comment:
xtensa?

Is this comment outdated?

https://github.com/zephyrproject-rtos/zephyr/blob/master/cmake/dts.cmake#L26

Reply:
Also, a reader might not be familiar with boarsd/posix.

So maybe instead say "Device tree is currently supported on all embedded targets".

Reply:
I found dts files in all extensa boards...

Reply:
> So maybe instead say "Device tree is currently supported on all embedded targets"

Well, it is supported by some qemu variants.
Sticking with posix and adding qemu_x86_64 whifch I initially forgot.